src/styles/base/_override.typography.scss

```diff
@@ -6,22 +6,37 @@
 .ds-h3,
 .ds-h4,
 .ds-h5,
-.ds-h6 {
+.ds-h6,
+.ds-text-heading--5xl,
```

Comment:
Added support for new classes https://cmsgov.github.io/mgov-design-system/styles/typography/